pytest_selenium/pytest_selenium.py

```diff
@@ -189,17 +197,10 @@ def pytest_addoption(parser):
                           '(default: %default)')
     group._addoption('--driver',
                      action='store',
+                     choices=SUPPORTED_DRIVERS,
```

Comment:
This will mean the case of the driver specified on the command line must exactly match the case of the choices defined. Are we okay with that? The previous code was more flexible about case.